lib/ortho_functions.py

```diff
@@ -530,6 +529,8 @@ def copy_to_wd(source_fp, wd):
             config.read(convert_windows_path(args.config_file))
             # Get the path from the config file
             gpkg_path = convert_windows_path(config.get("default", "gpkg_path"))
+            if platform.system() == "Windows" and "nunatak" in gpkg_path: # this is for selecting between specific versions of the reference DEM file
+                gpkg_path = gpkg_path.replace("nunatak","windows")
```

Comment:
This sort of directory path hard-coding is fragile in a publicly used tool.  Consider a solution that puts the system-specific login in the geopackage itself, like having two attributes, one for linux and one for Windows.  Let me know if you want to discuss the different approaches you can take.

Reply:
Also, shouldn't the [convert_windows_path](https://github.com/PolarGeospatialCenter/imagery_utils/blob/0a94f69e1298ae854d636bd01c2bb9b90b655ec0/lib/ortho_functions.py#L1955) function handle the dempath conversion without this?

Reply:
The convert_windows_path function handles the path to the .vrt (the DEM that gets passed to the ortho function) but the paths defined _within_ the .vrt are still system dependent, so we need to point to the different DEM .vrt for windows.